daemon/com.c

```diff
@@ -619,7 +620,7 @@ com_log_all_objects(
             {
                 if (object->thread_id_record_table[tid_index].lock_state != LOCK_STATE_NONE)
                 {
-                    log_message(MTC_LOG_DEBUG, "COM:     lock_state=%d thread_id=%x changed_time=%d(ms) .\n",
+                    log_message(MTC_LOG_DEBUG, "COM:     lock_state=%d thread_id=0x%lx changed_time=%"PRId64"(ms) .\n",
```

Comment:
I personally prefer the output of `%#lx` instead of `0x%lx`, `0x0` is ugly!

Reply:
`%#lx` works fine until it doesn't.  The moment you try `%#08x` or whatever, you end up with truncation.